Marlin/src/module/configuration_store.cpp

```diff
@@ -2520,38 +2534,42 @@ void MarlinSettings::reset(PORTARG_SOLO) {
           SERIAL_ECHOLNPGM_P(port, "Sensorless homing threshold:");
         }
         CONFIG_ECHO_START;
-        #ifdef X_HOMING_SENSITIVITY
-          #if ENABLED(X_IS_TMC2130) || ENABLED(IS_TRAMS)
-            say_M914(PORTVAR_SOLO);
-            SERIAL_ECHOLNPAIR_P(port, "X", stepperX.sgt());
-          #endif
-          #if ENABLED(X2_IS_TMC2130)
-            say_M914(PORTVAR_SOLO);
-            SERIAL_ECHOLNPAIR_P(port, "I1 X", stepperX2.sgt());
+        #define HAS_X_SENSORLESS (defined(X_HOMING_SENSITIVITY) && (ENABLED(X_IS_TMC2130) || ENABLED(IS_TRAMS)))
```

Comment:
Just letting you know that I'll be expanding the TMC related macros soon.
https://github.com/teemuatlut/Marlin/blob/bf2_tmc2660/Marlin/src/inc/Conditionals_post.h#L743

Reply:
Ooh, nice!